group.go

```diff
@@ -436,3 +446,36 @@ func (g *Group) Delete(recursive bool) error {
 	}
 	return nil
 }
+
+// AddMemberWithType adds a member to the Group providing its type
+// This method is recommended for performance when operating on remote groups
+func (g *Group) AddMemberWithType(uri, name string, isRelativeURI bool, objectType ObjectTypeEnum) error {
+	curi := C.CString(uri)
+	defer C.free(unsafe.Pointer(curi))
+
+	cname := C.CString(name)
+	defer C.free(unsafe.Pointer(cname))
+
+	var cRelative C.uint8_t
+	if isRelativeURI {
+		cRelative = 1
+	}
+
+	ret := C.tiledb_group_add_member_with_type(g.context.tiledbContext, g.group, curi, cRelative, cname, C.tiledb_object_t(objectType))
+	if ret != C.TILEDB_OK {
+		return fmt.Errorf("Error adding member with type to group: %s", g.context.LastError())
+	}
+	return nil
+}
+
+// IsOpen returns true if the Group is open or false if the group is closed
+func (g *Group) IsOpen() bool {
+	var isOpen C.int32_t
+
+	ret := C.tiledb_group_is_open(g.context.tiledbContext, g.group, &isOpen)
+	if ret != C.TILEDB_OK {
+		panic(fmt.Errorf("Error checking if group is open: %s", g.context.LastError()))
```

Comment:
I'm not a big fan of panicking here. We tend to only use panics if recovery is impossible. For example, REST panics if it cannot run database migrations upon starting. This doesn't strike me as a case where recovery is impossible, so I'd prefer to bubble up an error here.